pip/cmdoptions.py

```diff
@@ -119,6 +120,16 @@ def getname(n):
           ' levels).')
 )
 
+progress_bar = partial(
+    Option,
+    '--progress-bar',
+    dest='progress_bar',
+    type='choice',
+    choices=list(BAR_TYPES.keys()),
+    default='on',
+    help='Specify type of progress to be displayed [%s], '
+         '(default: %s)' % ('|'.join(BAR_TYPES.keys()), 'on'))
```

Comment:
You can directly pass `%default` in the help string, cf https://github.com/pypa/pip/blob/master/pip/cmdoptions.py#L222